andino_gz_classic/worlds/willowgarage.world

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" ?>
```

Comment:
Let's remove this file if it is not used.

Reply:
I thought it would be fine to have it just in case, but yeah I will remove it.